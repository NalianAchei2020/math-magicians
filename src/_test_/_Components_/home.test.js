import React from 'react';
import { render } from '@testing-library/react';

import Home from '../../components/home/home';

describe('Home component', () => {
  test('renders welcome message', () => {
    const { getByText } = render(<Home />);
    const welcomeMessage = getByText('Welcome to our page');
    expect(welcomeMessage).toMatchSnapshot();
  });
});
