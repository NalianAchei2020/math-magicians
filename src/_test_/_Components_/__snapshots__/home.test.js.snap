// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component renders welcome message 1`] = `
<h3>
  Welcome to our page
</h3>
`;
